docs/Makefile

```diff
@@ -61,8 +66,9 @@ apidocs:
 
 examples:
 	mkdir -p "${java_managed_examples}"
-	rsync -a --exclude-from=.examplesignore ../samples/ "${java_managed_examples}/"
-	rsync -a --exclude-from=.examplesignore ../samples/ "${spring_managed_examples}/"
+	rsync -a --exclude-from=.examplesignore ../samples/java-* "${java_managed_examples}/"
+	rsync -a --exclude-from=.examplesignore ../samples/scala-* "${java_managed_examples}/"
+	rsync -a --exclude-from=.examplesignore ../samples/spring-* "${spring_managed_examples}/"
```

Comment:
Java and Scala goes together on the java managed examples.

The Spring ones are apart.